packages/models/README.md

```diff
@@ -0,0 +1,186 @@
+# @craftercms/models
+
+This package contains data model definitions of different structures of Crafter CMS.
+
+## Usage
+
+- Install module using `yarn` or `npm`
+  - Yarn: `yarn add @craftercms/models`
+  - npm: `npm install @craftercms/models`
+- Import and use the models you need
+
+## Models
+---
+
+## - CrafterConfig
+Crafter configuration model for modules to work
+
+### Example
+
+- Defining an object of type Item:
+
+```js
+  import { CrafterConfig } from "@craftercms/models";
+
+  const config:CrafterConfig = {
+    "site": "editorial",
```

Comment:
Applies to all of these examples:
- This is JSON, remove quotes on prop name
- `?` is for interface declarations only
- Add space between the `:` and types (interface names in this case)
- Change the language annotations on code examples to the right language for each